### ISSUE TITLE
How to get y_predict when get B(variables for svm) in nonlinear svm

### ISSUE THREAD

**Issue by trangtv57** [nfmcclure/tensorflow_cookbook#97]
Sorry for my question, but I have something can't understand. I have some question about:

` rA = tf.reshape(tf.reduce_sum(tf.square(x_data), 1),[-1,1])
rB = tf.reshape(tf.reduce_sum(tf.square(prediction_grid), 1),[-1,1])
pred_sq_dist = tf.add(tf.subtract(rA, tf.multiply(2., tf.matmul(x_data, tf.transpose(prediction_grid)))), tf.transpose(rB))
pred_kernel = tf.exp(tf.multiply(gamma, tf.abs(pred_sq_dist)))

prediction_output = tf.matmul(tf.multiply(tf.transpose(y_target),b), pred_kernel)
prediction = tf.sign(prediction_output-tf.reduce_mean(prediction_output))
accuracy = tf.reduce_mean(tf.cast(tf.equal(tf.squeeze(prediction), tf.squeeze(y_target)), tf.float32))
`

When Ihave b (variable for SVM) get after train prcocess,  I can save it then use it for predict y. But If i use the same code like this (use pred_kernel) so calculatation prediction_output, I just can predict if i provide y target and get y_predict have shape same as y_target.
So I have aquestion some silly that if i have b( variable for svm) how can i calculate y_predict from just x[0] ( 1 x then predict 1 y). 
I am not know more about math, just want approve it for my app so sorry again.
Thanks you.

**Comment by nfmcclure** [comment]
Hi @trangtv57 ,

Thanks for your question.  There was a similar question before on how to predict on the non-linear SVM.  Because of this, if you look at the jupyter notebook:
https://github.com/nfmcclure/tensorflow_cookbook/blob/master/04_Support_Vector_Machines/05_Implementing_Nonlinear_SVMs/05_nonlinear_svm.ipynb

Near the bottom of the file, there is a section called, "Evaluate Test Points", that does prediction on input points.  Please review this section and the code involved to see how to predict on new points.

I hope that helps.

**Comment by trangtv57** [comment]
thanks you guys.
